ckanext/opendatani/helpers.py

```diff
@@ -179,8 +179,9 @@ def prepare_csv_report(resource):
     :return: a string containing the csv_id of the created archive
     :rtype: string
     """
-    if toolkit.c.user is None:
-        return None
+    if not toolkit.c.user or not resource:
```

Comment:
Can you give me a case when user is not there or `not resource`? Not sure why we need this if statement

Reply:
I think at one point it was possible that I was passing None as a resource, but I seem to have fixed it elsewhere already.